jcommune-model/src/main/resources/org/jtalks/jcommune/model/sample-forum.sql

```diff
@@ -1,32 +1,67 @@
-﻿SET @forum_component_id := 2;
```

Comment:
Why you removed this line? On newly created database component ID will be undefined.